e2e-tests/native/test/Action.ts

```diff
@@ -0,0 +1,74 @@
+type SwipeDirectionType = 'UP' | 'DOWN'
+
+type SwipePoint = { x: number, y: number }
+
+const swipeMove = (startPoint: SwipePoint, endPoint: SwipePoint): Array<{ action: string, options: any }> => {
+    return [
+        {
+            action: 'press',
+            options: startPoint,
+        },
+        {
+            action: 'wait',
+            options: {
+                ms: 200,
+            },
+        },
+        {
+            action: 'moveTo',
+            options: endPoint,
+        },
+        {
+            action: 'release',
+            options: {},
+        }
+    ]
+}
+
+export const swipe = async (direction: SwipeDirectionType, repeat: number = 1, amount: number = 80) => {
```

Comment:
I get an eslint error here for 'no-inferrable-types'. Is not linting done for these files here?